docs/semgrep-ci/overview.md

```diff
@@ -49,27 +52,37 @@ If you’re already running [GitLab SAST](https://docs.gitlab.com/ee/user/applic
 
 ### Other CI providers
 
-To add Semgrep CI to any CI environment, use the [`returntocorp/semgrep-agent:v1`](https://hub.docker.com/r/returntocorp/semgrep-agent) Docker image directly:
+The community successfully run Semgrep CI on the following platforms (some include a link to sample configuration):
 
-For full project scans:
+- Bitbucket Pipelines
+- Bitrise
+- Buildbot
+- Buildkite [(sample configuration)](../sample-ci-configs/#buildkite)
+- CircleCI [(sample configuration)](../sample-ci-configs/#circleci) 
+- Codefresh
+- Jenkins [(sample configuration)](../sample-ci-configs/#jenkins)
+- TeamCity CI
+- Travis CI
 
-```sh
-docker run -v $(pwd):/src --workdir /src returntocorp/semgrep-agent:v1 semgrep-agent --config auto --config <other rule or rulesets>
-```
+To add Semgrep CI to any CI environment, use the [`returntocorp/semgrep`](https://hub.docker.com/r/returntocorp/semgrep) Docker image directly.
 
-Set the `--baseline-ref` flag to the git ref (branch name, tag, or commit hash) to use it as a baseline. Semgrep will scan only the files modified in your branch and output the difference in findings between the baseline branch and the new branch. If you are using Semgrep App, it will also close all prior findings on that branch that it no longer finds, treating them as fixed. Therefore, doing full scans and diff scans on the same branch is not recommended. For example, to report findings newly added since branching off from your `main` branch, run
+For full project scans:
 
 ```sh
-semgrep-agent --baseline-ref main
+docker run -v $(pwd):/src --workdir /src returntocorp/semgrep semgrep ci --config auto --config <other rule or rulesets>
```

Comment:
Is this still valid? @brendongo @underyx

Reply:
Yes but `semgrep ci --config auto` also works (without using docker) and is more equivalent to the line you are replacing

Reply:
![Screenshot 2022-04-08 at 12 10 02](https://user-images.githubusercontent.com/26669307/162415115-c3e0494d-55cb-462e-b7d2-660ab82a4aba.jpg)
There is no line being replaced actually... I just deleted a baseline mentioned above, because we explain it in a different article. I have not touched this command...

Reply:
@brendongo why somebody would like to use such a long command? Maybe this comes from some older version of the agent?

Reply:
It was because you could not pip install semgrep-agent so had to use docker

Reply:
@brendongo @underyx Substituted with suggested text. Please check it. This is under a "Other CI providers" section in the Semgrep CI overview document.

```
For full project scans:

```sh
semgrep ci --config auto --config <other rule or rulesets>
```

**Note**: If you need to use a different image than docker, install Semgrep CI by `pip install semgrep`.
```